rhizome/host/lib/cert_server_setup.rb

```diff
@@ -48,10 +48,7 @@ def vm_server_path
   end
 
   def package_url
-    Arch.render(
-      x64: "https://github.com/ubicloud/metadata-endpoint/releases/download/v#{server_version}/metadata-endpoint_Linux_x86_64.tar.gz",
-      arm64: "https://github.com/ubicloud/metadata-endpoint/releases/download/v#{server_version}/metadata-endpoint_Linux_arm64.tar.gz"
-    )
+    "https://github.com/ubicloud/metadata-endpoint/releases/download/v#{server_version}/metadata-endpoint_Linux_#{Arch.x64? ? "x86_64" : "arm64"}.tar.gz"
```

Comment:
IMO: I don't like this conversion, where "else" assumes some architecture. `Arch.render` is more reliable to grep and had exhaustiveness checking. I could abide it, but previously, I forbade it.

Reply:
It's still easy to grep for `Arch.`.  I can understand the point of exhaustiveness checking, if you expect to move beyond 2 architectures (if not, YAGNI).  Anyway, if you would forbade the approach if someone else suggested it, I should not be special, I'll switch to an alternative approach.

Reply:
I suppose grepping for all `Arch` is a solution, mostly, I like that grepping for `Arch.render` can identify cases that are intended exhaustive to be exhaustive, rather than `Arch.x64?` and `Arch.arm64?`, which I anticipated -- unrealized, I think -- other forms of use (when needing to run different imperative code around the branch, or when one architecture requires a special action, e.g. `do_this if Arch.arm64?`. But selecting among two objects has been enough for everything, which I did not expect. If I knew that from the outset, maybe I would have done things the normal way.